packages/@pollyjs/adapter-xhr/package.json

```diff
@@ -40,9 +40,10 @@
     "watch-all": "npm-run-all --parallel build:watch test:build:watch"
   },
   "dependencies": {
+    "@offirgolan/nise": "^4.1.0",
```

Comment:
Using a user scoped package with sinonjs/nise#164 until it gets merged upstream.